tests/nwchem_tests/test_eom_ccsd_h2o.py

```diff
@@ -75,17 +75,19 @@ def check_eomccsd(return_value):
 def test_1_eomccsd():
     qcdb.set_options({
         'basis'     :       '6-31g*',
+        'memory'    :       '15000 mb',
```

Comment:
might try reverting some of these memory lines and make sure still run. not all computers have 15gb

Reply:
Changed back all the memory values in NWChem tests with recent two commits (distinguish between passing/not-passing tests)